app/components/Panel.js

```diff
@@ -100,12 +105,9 @@ export default class Panel extends Component {
   }
 
   selectedApolloLog() {
-    const logIsPopulated = this.state.actionLog && this.state.actionLog.length;
-
-    if (!logIsPopulated) {
-      return null;
+    if (this.state.actionLog.length < 1) {
+      this.state.actionLog[0] = {};
```

Comment:
Let's make sure not to mutate the `state` of the component directly - let's just `return {}` here.